plugins/mac-capture/mac-screen-capture.m

```diff
@@ -377,9 +377,30 @@ static bool init_screen_stream(struct screen_capture *sc)
 				 initWithDisplay:target_display
 				excludingWindows:[[NSArray alloc] init]];
 		} else {
+			__block SCRunningApplication *excluded;
+			[sc->shareable_content.applications
```

Comment:
Isn't `applications` already an NSArray? Instead of enumerating, extracting a single item and then creating a new NSArray with that single item, `NSArray` should have a block-based filter that returns an NSArray with just the item you want.

Then you can pass this filtered copy of the NSArray directly into the next call.